WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -414,6 +417,14 @@ public static void addNewPageForResult(@NonNull Fragment fragment, @NonNull Site
         fragment.startActivityForResult(intent, RequestCodes.EDIT_POST);
     }
 
+    public static void viewRevisionDetailsForResult(Activity activity, Revision revision,
```

Comment:
```suggestion
    public static void viewHistoryDetailForResult(Activity activity, Revision revision,
```

Can we rename this method `viewHistoryDetailForResult` for consistency?